packages/blockchain-wallet-v4-frontend/src/data/components/refresh/sagas.ts

```diff
@@ -4,7 +4,37 @@ import { call, put, select } from 'redux-saga/effects'
 import { actions, selectors } from 'data'
 
 export default () => {
-  const refreshClicked = function * () {
+  const refreshBchTransactions = function* () {
+    const onlyShow = yield select(selectors.components.bchTransactions.selectOnlyShow)
+    yield put(actions.core.data.bch.fetchTransactions(onlyShow, true))
+  }
+
+  const refreshBtcTransactions = function* () {
```

Comment:
Isn't there a way to make it more generic? for the erc20 that will come?

Reply:
const refreshErc20Transactions = function* (coin) {
    yield put(actions.core.data.eth.fetchErc20Transactions(coin, true))
  }

already pretty generic, other coins will need to be refactored later...